src/components/gui/gui.jsx

```diff
@@ -119,6 +119,6 @@ GUIComponent.propTypes = {
     vm: PropTypes.instanceOf(VM).isRequired
 };
 GUIComponent.defaultProps = {
-    basePath: '/'
+    basePath: './'
```

Comment:
Not strictly, but useful since it allows updating the location of the scratch-blocks media if the directory structure changes.